yarn.lock

```diff
@@ -10412,6 +10412,15 @@ postcss@^8.2.15, postcss@^8.3.11, postcss@^8.4.5:
     picocolors "^1.0.0"
     source-map-js "^1.0.1"
 
+postcss@^8.4.7:
+  version "8.4.12"
+  resolved "https://registry.yarnpkg.com/postcss/-/postcss-8.4.12.tgz#1e7de78733b28970fa4743f7da6f3763648b1905"
```

Comment:
all the new packages are pulling from `yarnpkg.com`, and the status quo links are all `npmjs.com`... i guess for consistency the `npmjs` is better?

maybe reset to the starting point, `jlpm cache clean` to be sure?

Reply:
Indeed it looks like both were mixed up. https://github.com/jupyterlab/jupyterlab/pull/12350/commits/4eabb6e1b45d89baa3ef2af995abce0d553e4bfe normalized on `npmjs.org`.

https://github.com/jupyterlab/jupyterlab/pull/12350/commits/b7ce2f402730c8bcb89e15a7a81c64d9e86f0a6b tried to add the default registry to `.yarnrc` but it somehow made the `release_test` check fail.

So we can defer that to a separate issue / PR since it's not strictly related to this change.